src/BuildAfterTargetingPack/BuildAfterTargetingPack.csproj

```diff
@@ -76,19 +76,19 @@
     </MSBuild>
   </Target>
```

Comment:
should `GetReferencesProvidedDelayedProjects` also be filtered out for source build?